### ISSUE TITLE
临时缓常报错，需要自己手动在temp创建ckpts文件夹，两个模型也要手动下载。

### ISSUE THREAD

**Issue by NGwr** [Fannovel16/comfyui_controlnet_aux#478]
1、Failed to find D:\database\ai\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts\yzd-v/DWPose\yolox_l.onnx.
Downloading from huggingface.co
ComfyUI未能找到yolox_l.onnx模型文件，并尝试从Hugging Face下载。

2、下载过程中出现了路径错误，C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\ckpts路径不存在。
缓存目录为C:\Users\ADMINI~1\AppData\Local\Temp

cacher folder is C:\Users\ADMINI~1\AppData\Local\Temp, you can change it by custom_tmp_path in config.yaml
[WinError 3] 系统找不到指定的路径。: 'C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\ckpts'

3、Failed to find D:\database\ai\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts\hr16/DWPose-TorchScript-BatchSize5\dw-ll_ucoco_384_bs5.torchscript.pt.
Downloading from huggingface.co

后台日志：
got prompt
model weight dtype torch.float16, manual cast: None
model_type EPS
Using xformers attention in VAE
Using xformers attention in VAE
Failed to find D:\database\ai\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts\yzd-v/DWPose\yolox_l.onnx.
 Downloading from huggingface.co
cacher folder is C:\Users\ADMINI~1\AppData\Local\Temp, you can change it by custom_tmp_path in config.yaml
D:\database\ai\ComfyUI\python\lib\site-packages\huggingface_hub\file_download.py:1194: UserWarning: `local_dir_use_symlinks` parameter is deprecated and will be ignored. The process to download files to a local folder has been updated and do not rely on symlinks anymore. You only need to pass a destination folder as`local_dir`.
For more details, check out https://huggingface.co/docs/huggingface_hub/main/en/guides/download#download-files-to-local-folder.
  warnings.warn(
[WinError 3] 系统找不到指定的路径。: 'C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\ckpts'
model_path is D:\database\ai\ComfyUI\.cache\sdwebuilauncher\hfmirror\refs\yzd-v\DWPose\main\yolox_l.onnx
Failed to find D:\database\ai\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts\hr16/DWPose-TorchScript-BatchSize5\dw-ll_ucoco_384_bs5.torchscript.pt.
 Downloading from huggingface.co
cacher folder is C:\Users\ADMINI~1\AppData\Local\Temp, you can change it by custom_tmp_path in config.yaml
[WinError 3] 系统找不到指定的路径。: 'C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\ckpts'
model_path is D:\database\ai\ComfyUI\.cache\sdwebuilauncher\hfmirror\refs\hr16\DWPose-TorchScript-BatchSize5\main\dw-ll_ucoco_384_bs5.torchscript.pt

DWPose: Using yolox_l.onnx for bbox detection and dw-ll_ucoco_384_bs5.torchscript.pt for pose estimation
DWPose: Caching ONNXRuntime session yolox_l.onnx...

DWPose: Caching TorchScript module dw-ll_ucoco_384_bs5.torchscript.pt on ...
DWPose: Bbox 308.36ms
DWPose: Pose 552.05ms on 1 people

Requested to load SD1ClipModel
Loading 1 new model
loaded completely 0.0 235.84423828125 True
D:\database\ai\ComfyUI\comfy\ldm\modules\attention.py:407: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)
  out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False)
Requested to load BaseModel
Requested to load ControlNet
Loading 2 new models
loaded completely 0.0 1639.406135559082 True
loaded completely 0.0 689.0852355957031 True
Requested to load AutoencoderKL
Loading 1 new model
